runtime/lang/menu_fi_fi.latin1.vim

```diff
@@ -178,7 +178,7 @@ menutrans &Get\ Block					&Hae\ lohko
 menutrans &Put\ Block					&Lis��\ lohko
 
 " -SEP2-
-menutrans &Make<Tab>:make			&Make<Tab>:make
```

Comment:
I think make here refers to kind of a make command of a build system and translating it would be just confusing.